engine/release-notes/index.md

```diff
@@ -22,6 +22,50 @@ for Docker Engine.
 
 # Version 20.10
 
+## 20.10.6
+2021-04-12
+
+### Client
+
+* Apple Silicon (darwin/arm64) support for Docker CLI [docker/cli#3042](https://github.com/docker/cli/pull/3042)
+* config: print deprecation warning when falling back to pre-v1.7.0 config file `~/.dockercfg`. Support for this file will be removed in a future release [docker/cli#3000](https://github.com/docker/cli/pull/3000)
+
+### Builder
+
+* Fix classic builder silently ignoring unsupported Dockerfile options and prompt to enable BuildKit instead [moby/moby#42197](https://github.com/moby/moby/pull/42197)
+
+### Logging
+
+* json-file: fix sporadic unexpected EOF errors [moby/moby#42174](https://github.com/moby/moby/pull/42174)
+
+### Networking
+
+* Fix a regression in docker 20.10, causing  IPv6 addresses no longer to be bound by default when mapping ports [moby/moby#42205](https://github.com/moby/moby/pull/42205)
+* Fix implicit IPv6 port-mappings not included in API response. Before docker 20.10, published ports were accessible through both IPv4 and IPv6 by default, but the API only included information about the IPv4 (0.0.0.0) mapping [moby/moby#42205](https://github.com/moby/moby/pull/42205)
+* Fix a regression in docker 20.10, causing the docker-proxy  to not be terminated in all cases [moby/moby#42205](https://github.com/moby/moby/pull/42205)
+* Fix iptables forwarding rules not being cleaned up upon container removal [moby/moby#42205](https://github.com/moby/moby/pull/42205)
+
+### Packaging
+
+* Update containerd to [v1.4.4](https://github.com/containerd/containerd/releases/tag/v1.4.4) for static binaries. The containerd.io package on apt/yum repos already had this update out of band. Includes a fix for [CVE-2021-21334](https://github.com/containerd/containerd/security/advisories/GHSA-6g2q-w5j3-fwh4). [moby/moby#42124](https://github.com/moby/moby/pull/42124)
+* Include VPNKit binary for arm64 [moby/moby#42141](https://github.com/moby/moby/pull/42141)
+
```

Comment:
```suggestion
* Add the [Docker Scan CLI](https://github.com/docker/scan-cli-plugin) plugin

```

Reply:
We should mention it's currently x86 only

I earlier wrote;

- Provide docker scan CLI plugin through the `docker-scan-plugin` package on Linux (x86)
- Add docker scan plugin as "recommended" dependency for the `docker-ce-cli`  package on Linux (x86)

But perhaps that's too technical; wdyt, @chris-crone ?

Reply:
Yes, it'd be good to say x86 only as that impacts users

Reply:
```
* Provide the [Docker Scan CLI](https://github.com/docker/scan-cli-plugin) plugin on Linux amd64 via a `docker-scan-plugin` package as a recommended dependency for the `docker-ce-cli` package [docker/docker-ce-packaging#537](https://github.com/docker/docker-ce-packaging/pull/537)
```